Client/Ecosia/UI/Onboarding/WelcomeTourTransparent.swift

```diff
@@ -0,0 +1,115 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+import Core
+
+final class WelcomeTourTransparent: UIView, NotificationThemeable {
+
+    private weak var stack: UIStackView!
+
+    lazy var currencyNumberFormatter: NumberFormatter = {
+        let formatter = NumberFormatter()
+        formatter.numberStyle = .currency
+        formatter.currencySymbol = "€"
+        return formatter
+    }()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+        updateAccessibilitySettings()
+    }
+
+    required init?(coder: NSCoder) {  nil }
+
+    func setup() {
+        let stack = UIStackView()
+        stack.translatesAutoresizingMaskIntoConstraints = false
+        stack.axis = .vertical
+        stack.distribution = .fill
+        stack.alignment = .leading
+        stack.spacing = 8
+        addSubview(stack)
+        self.stack = stack
+
+        stack.leadingAnchor.constraint(equalTo: leadingAnchor).isActive = true
+        stack.trailingAnchor.constraint(equalTo: trailingAnchor).isActive = true
+        stack.topAnchor.constraint(equalTo: topAnchor, constant: 54).isActive = true
+        let height = heightAnchor.constraint(equalToConstant: 200)
+        height.priority = .init(rawValue: 500)
+        height.isActive = true
+        
+        addMonthView(toStack: stack)
+        
+        let financialReports = TreeCounter.shared.statistics.financialReports
+        if let totalIncome = currencyNumberFormatter.string(from: .init(value: financialReports.value)) {
+            let income = WelcomeTourRow(image: "financialReports", title: totalIncome, text: .localized(.totalIncome))
+            stack.addArrangedSubview(income)
+        }
+
+        let treesFinanced = "741,333" // TODO: Read from somewhere dynamically
```

Comment:
⚠️ 😇 ❗ huger TODO! ahha

Reply:
Indeed! This and analytics are the reasons this is only draft for now, soon to be fixed 🚀